configs/AM437X/AM437X_linux_toc.txt

```diff
@@ -72,6 +66,10 @@ linux/Foundational_Components_Kernel_Release_Notes
 linux/Foundational_Components_Kernel_RT_Release_Notes
 linux/Foundational_Components_Kernel_LTP-DDT_Validation
 linux/Foundational_Components_Kernel_FAQs
+linux/Foundational_Components/Power_Management/pm_overview
+linux/Foundational_Components/Power_Management/pm_dvfs
+linux/Foundational_Components/Power_Management/pm_cpuidle
+linux/Foundational_Components/Power_Management/pm_suspend_resume
```

Comment:
@cshilwant In the preview, can't find the Power Management section in Foundational Components. But able to open the page using search functionality, and it is not highlighted in Table of Contents on the left side.
See - https://cshilwant.github.io/processor-sdk-doc/processor-sdk-linux-AM335X/esd/docs/09_03_05_02/linux/Foundational_Components/Power_Management/pm_suspend_resume.html

Reply:
Same for AM335x_linux_toc

Reply:
Good catch @jeevantelukula 

Updated the PR & I now see it in Table of contents - https://cshilwant.github.io/processor-sdk-doc/processor-sdk-linux-AM335X/esd/docs/09_03_05_02/linux/Foundational_Components.html#